aws/data_source_aws_elasticsearch_domain.go

```diff
@@ -283,6 +299,10 @@ func dataSourceAwsElasticSearchDomainRead(d *schema.ResourceData, meta interface
 	d.Set("endpoint", ds.Endpoint)
 	d.Set("kibana_endpoint", getKibanaEndpoint(d))
 
+	if err := d.Set("advanced_security_options", flattenAdvancedSecurityOptions(ds.AdvancedSecurityOptions)); err != nil {
+		return fmt.Errorf("error setting advanced_security_options: %s", err)
```

Comment:
Nit: do you mind switching this to the new error wrapping in Go 1.13 ("%w"), rather than the string value of the original error?